docs/v3.x/getting-started/ruby.md

```diff
@@ -0,0 +1,273 @@
+# Getting Started with Ruby
+
+This integration guide is following the [Getting started guide](../getting-started/quick-start.html). We assume that you have completed [Step 8](../getting-started/quick-start.html#_8-consume-the-content-type-s-api) and therefore can consume the API by browsing this [url](http://localhost:1337/restaurants).
+
+If you haven't gone through the getting started guide, the way you request a Strapi API with [Ruby](https://www.ruby-lang.org/en/) remains the same except that you will not fetch the same content.
+
+### Create a Ruby file
+
+Be sure to have Ruby installed on your computer
+
+```bash
+touch script.rb
+```
+
+### Use an HTTP client
+
+Many HTTP clients are available but in this documentation we'll use [HTTParty](https://github.com/jnunemaker/httparty)
+
+```bash
+gem install httparty
+```
+
+### GET Request your collection type
+
+Execute a GET request on the `restaurant` Collection Type in order to fetch all your restaurants.
+
+Be sure that you activated the `find` permission for the `restaurant` Collection Type.
+
+_Request_
+
+```ruby
+HTTParty.get('http://localhost:1337/restaurants',
+  header: {
+    'Content-Type' => 'application/json'
+  })
+```
+
+_Response_
+
+```json
+[
+  {
+    "id": 1,
+    "name": "Biscotte Restaurant",
+    "description": "Welcome to Biscotte restaurant! Restaurant Biscotte offers a cuisine based on fresh, quality products, often local, organic when possible, and always produced by passionate producers.",
+    "created_by": {
+      "id": 1,
+      "firstname": "Paul",
+      "lastname": "Bocuse",
+      "username": null
+    },
+    "updated_by": {
+      "id": 1,
+      "firstname": "Paul",
+      "lastname": "Bocuse",
+      "username": null
+    },
+    "created_at": "2020-07-31T11:37:16.964Z",
+    "updated_at": "2020-07-31T11:37:16.975Z",
+    "categories": [
+      {
+        "id": 1,
+        "name": "French Food",
+        "created_by": 1,
+        "updated_by": 1,
+        "created_at": "2020-07-31T11:36:23.164Z",
+        "updated_at": "2020-07-31T11:36:23.172Z"
+      }
+    ]
+  }
+]
+```
+
+### Example
+
+```ruby
```

Comment:
Can you describe the file structure of this app?

I think we need a Gemfile as well to install `gems` so they are not installed globally.

Reply:
Can you make sure this app follows the same views as the others so all exemples are consistent?

Reply:
I think the less configuration file there are the easier it will be for the user so just installing the gem globally is a good thing instead of having a Gemfile installed etc...

Reply:
Actually as developer point of view I really don't like to install gems or modules globally.